Assets/MixedRealityToolkit.ThirdParty/MRTK-Quest/Scripts/Input/Controllers/OculusQuestHand.cs

```diff
@@ -146,6 +146,10 @@ public override bool IsInPointingPose
 
         protected bool IsPinching { set; get; }
 
+        // Pinch was also used as grab, we want to allow hand-curl grab not just pinch.
+        // Determine pinch and grab separately
```

Comment:
Ideally comments like this are done in summar blocks, as the ones before are written. This isn't a public field however, so not a huge deal

Reply:
yeah I didn't think that was necessary for non-public (e.g. there is no comment for IsPinching here)

Reply:
Yeah it's not strictly necessary, but having a summary block makes it easier to refactor a bit later if need be. Again though, minor nit, and you don't have to change this for me to accept it in.